components/collective-page/hero/Hero.js

```diff
@@ -67,11 +68,25 @@ const StyledShortDescription = styled.h2`
 const Hero = ({ collective, host, isAdmin, onPrimaryColorChange, callsToAction, intl }) => {
   const [hasColorPicker, showColorPicker] = React.useState(false);
   const [isEditingCover, editCover] = React.useState(false);
+  const [hasMessage, showMessage] = React.useState(false);
```

Comment:
`hasMessage` makes it sounds like it is a boolean. Can we use just `message`?

```suggestion
  const [message, showMessage] = React.useState(null);
```

Reply:
Understood.